### ISSUE TITLE
Observable switch operator is incorrectly typed in NPM package

### ISSUE THREAD

**Issue by viking** [ReactiveX/rxjs#2811]
**RxJS version:**
5.4.3

**Code to reproduce:**
```
import { Observable } from 'rxjs/Observable';
import 'rxjs/add/operator/switch';
import 'rxjs/add/observable/of';

Observable.of("foo").switch().subscribe(foo => console.log(foo));
```

**Expected behavior:**
Code has no errors.

**Actual behavior:**
test.ts(5,31): error TS2339: Property 'subscribe' does not exist on type 'string'.

**Additional information:**
This only happens from the npm installed version of rxjs. The switch operator has the wrong type in the file `rxjs/operator/switch.d.ts`:

```
export declare function _switch<T>(this: Observable<T>): T;
```

**Comment by viking** [comment]
Since this is fixed in master, I'll close this, but I didn't find any closed bug report on this so I thought it would be a good idea for anyone who had this problem in the future to find.

**Comment by viking** [comment]
This was fixed by commit 2f12572ec5f65b68c9d3c5b29e36383def6d888c.
